gbpservice/nfp/service_plugins/vpn/drivers/nfp_vpnaaas_driver.py

```diff
@@ -0,0 +1,418 @@
+import socket
+
+from neutron.common import exceptions
+from neutron.common import rpc as n_rpc
+from neutron.db import agents_db
+from neutron.db import agentschedulers_db
+from neutron import manager
+from neutron_vpnaas.db.vpn import vpn_validator
+from neutron_vpnaas.services.vpn.common import constants as const
+from neutron_vpnaas.services.vpn.common import topics
+from neutron_vpnaas.services.vpn.plugin import VPNPlugin
+from neutron_vpnaas.services.vpn import service_drivers
+from neutron_vpnaas.services.vpn.service_drivers import base_ipsec
+
+from oslo_log import log as logging
+import oslo_messaging
+
+LOG = logging.getLogger(__name__)
+
+BASE_IPSEC_VERSION = '1.0'
```

Comment:
The name is suggesting the type of VPN. Can this be suggesting the service type, like BASE_VPN_VERSION